libs/kameleon/src/kameleon/entities.clj

```diff
@@ -298,3 +298,6 @@
 (defentity user-saved-searches
   (table :user_saved_searches)
   (belongs-to users {:fk :user_id}))
+
+(defentity job-status-updates
+  (table :job_status_updates))
```

Comment:
I haven't really used korma much but it seems like the link via the `external_id` column can also be added here in some sort of productive fashion.